### PR TITLE
add astro airflow pause

### LINKED ISSUE #18: Proper naming of Pause, Kill, Stop Commands
I propose that we use two commands for these behaviors

`astro airflow stop` 
A non-destructive wrapper around [docker stop](https://docs.docker.com/engine/reference/commandline/stop/) which maintains mounted volumes. Similar to how some may view a pause command

`astro airflow kill`
A destructive tear down of containers. How `astro airflow stop` currently works at time of issue creation.

### PR DESCRIPTION
**Pull request by andscoop**
Closes #18 and #5

### CODE REVIEW

File: airflow/docker.go [hunk 1]
```diff
@@ -85,8 +85,8 @@ func generateConfig(projectName, airflowHome string) string {
 	return buff.String()
 }
 
-// createProjectFromContext creates project with yaml config as context
-func createProjectFromContext(projectName, airflowHome string) (project.APIProject, error) {
+// fetchDockerProject creates project with yaml config as context
+func fetchDockerProject(projectName, airflowHome string) (project.APIProject, error) {
```
Comment (andscoop) [inline airflow/docker.go]: still not sure about the naming of this func

File: airflow/docker.go [hunk 2]
```diff
@@ -107,30 +107,50 @@ func Start(airflowHome string) error {
...
+	// Fetch project containers
+	psInfo, err := project.Ps(context.Background())
 	if err != nil {
-		return errors.Wrap(err, "Error building, (re)creating or starting project containers")
+		return errors.Wrap(err, "Error checking docker-compose status")
+	}
+
+	if len(psInfo) > 0 {
+		// Ensure project is not alreasdy running
+		for _, info := range psInfo {
```
Comment (andscoop) [inline airflow/docker.go]: Prevents a user from running start and having a hard error on an already running project.

File: airflow/docker.go [hunk 3]
```diff
@@ -107,30 +107,50 @@ func Start(airflowHome string) error {
...
+			if info["State"][0:2] == "Up" {
```
Comment (schnie) [inline airflow/docker.go]: Wonder if there's a constant for "Up" in `libcompose`. If not, oh well.

File: airflow/docker.go (hunk 1 above)
Comment (schnie) [reply airflow/docker.go]: Yea, how about just `createProject`?

**Comment by andscoop** [comment]
@schnie made requested changes This should be good to go

**Pull request merged**